content/conditions/cold/panel-2-main.md

```diff
@@ -0,0 +1,6 @@
+## See someone urgently if  
```

Comment:
We should move this to a urgent callout rather than a panel after speaking to the content team. The panel pattern doesn't fit the content we're using in this instance. "Someone" is too vague in this instance and we should use the urgent heading of "See your GP now or go to A&E if".